extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -275,13 +296,21 @@ export class AmpAd3PImpl extends AMP.BaseElement {
       user().assert(!this.isInFixedContainer_,
           '<amp-ad> is not allowed to be placed in elements with ' +
           'position:fixed: %s', this.element);
+      /** detect ad containers, add the list to element as a new attribute */
+      if (!this.element.getAttribute('data-amp-container-element')) {
```

Comment:
This shouldn't be a `data` attribute, since that would allow the user to specify it. Ie. I can make a sticky ad report as a normal ad.